### ISSUE TITLE
Breaks AWS Lambda

### ISSUE THREAD

**Issue by dncrews** [MauriceButler/bunyan-loggly#24]
As of #16 (v1.3), this library is no longer usable in AWS Lambda. Because that library waits for 30 seconds before sending ANY logs, it has broken all of my Lambda calls that go through API Gateway, which times out at 30 seconds. This library now causes my entire stack to be locked up and always respond with timeouts.

**Comment by dncrews** [comment]
Looks like this is fixed in version 2.2.0 of node-loggly-bulk. Can you update, please?

**Comment by MauriceButler** [comment]
Hi @dncrews 

As per #23 there is a new version of node-loggly-bulk about to be released that also fixes another issue stopping logging from occurring. 

As soon as that is published Ill update and republish myself.

**Comment by MauriceButler** [comment]
Updated to node-loggly-bulk@2.2.1 with 2b44f10760f8c24be297597d252c240e2fbf74d0

Version 1.3.3 now published which should resolve this.

**Comment by dncrews** [comment]
It didn't.

**Comment by MauriceButler** [comment]
Can you please provide the version you are using and how you are creating and using the logger?

**Comment by dncrews** [comment]
bunyan-loggly: v1.3.4

```js
const bunyan = require('bunyan');
const Bunyan2Loggly = require('bunyan-loggly');
const config = {
  subdomain: '<subdomain>',
  token: '<token>',
  enabled: true,
  useTagHeader: false,
  tags: [ '<tag1>', '<tag2>' ],
  level: 'debug'
};
const bufferLength = 5;
const bufferTimeout = 1000;
const logglyStream = new Bunyan2Loggly(config, bufferLength, bufferTimeout);
const streams = [ logglyStream, whateverStream ];

const logger = bunyan.createLogger({
  level: 'debug',
  name: '<whatever>',
  streams: streams,
  serializers: serializers
});

logger.info('something');
```

#### Expected result
It should wait for 1 second and then send the logs.

#### Actual result
It waits for 31 seconds and then sends the logs. As the API endpoint will not respond until all processes are finished, all API requests timeout after 30s.

**Comment by dncrews** [comment]
@MauriceButler we spent some time debugging the "why" of this one. It looks like you're managing your own queue for bulking and THEN using the node-loggly-bulk to have its own queue for bulking, too. https://github.com/MauriceButler/bunyan-loggly/pull/27 is the result of that debugging, which removes the custom queuing in this library and only uses the queue and bulk of node-loggly-bulk without changing your API.

**Comment by MauriceButler** [comment]
Thanks @dncrews and @ziplizard

Ill have a look through the PR now.

**Comment by MauriceButler** [comment]
Should be fixed in [v1.3.5](https://github.com/MauriceButler/bunyan-loggly/releases/tag/v1.3.5)

Please let me know if there is still issues.
